src/clip.sol

```diff
@@ -449,6 +449,7 @@ contract Clipper {
         (done, price) = status(tic, sales[id].top);
 
         needsRedo = usr != address(0) && done;
+        lot = sales[id].lot;
```

Comment:
Would it be interesting to return `tab` as well? (Since that determines the maximum amount of collateral that can be bought.)

Or maybe we should just return `max(lot, tab / price)`?

Reply:
I think we should just return the `tab` too.

Reply:
This is right, open question whether we want `lot` and `tab` or just the amount of collateral available at that price.

Reply:
so `function getStatus(uint256 id) external view returns (bool needsRedo, uint256 price, uint256 lot, uint256 tab)`

Reply:
Added tab return

Reply:
Yeah, all three of `tab/lot/price` makes the most sense, as then Keepers can just do whatever calculations they want, including the one I described above.